reference/forms/types/range.rst

```diff
@@ -0,0 +1,61 @@
+.. index::
+   single: Forms; Fields; range
+
+range Field Type
+================
+
+The ``range`` field is a slider that is rendered using the HTML5
+``<input type="range" />`` tag.
+
++-------------+---------------------------------------------------------------------+
+| Rendered as | ``input`` ``range`` field (slider in HTML5 supported browser)       |
++-------------+---------------------------------------------------------------------+
+| Inherited   | - `data`_                                                           |
+| options     | - `disabled`_                                                       |
+|             | - `empty_data`_                                                     |
```

Comment:
Hey @snoek09 actually I have a question regarding the inherited options and include. I actually copied this from `email.rst` or something I guess. So is it all the properties of the text field for it is parent the properties are imported ?